.github/dependabot.yaml

```diff
@@ -1,14 +1,34 @@
 version: 2
 updates:
+  # GitHub Actions updates (weekly)
   - package-ecosystem: github-actions
-    directory: "/"
+    directory: "/.github/"
     schedule:
       interval: "weekly"
       day: "sunday"
-    open-pull-requests-limit: 10
+      time: "08:00"
     rebase-strategy: disabled
+    labels:
+      - "dependencies"
+      - "automated"
+    reviewers:
+      - "soumeh01"
```

Comment:
do we need an additional person to avoid a single point of failure?

Reply:
Yes, Ideal case would be a primary stakeholder & secondary go-to person for the repo.

Reply:
[Added another reviewer](https://github.com/Open-CMSIS-Pack/cbuild/pull/378/commits/adbdfe3c88ef839bdd416b069a15706df999907a)